webui/src/styles/_variables.scss

```diff
@@ -1,14 +1,14 @@
 // _variables.scss
 
 $bgd-color: #fff8f8;
-$primary-blue: #001c70;
+$primary-blue: #05f;
```

Comment:
What is the reason for the colour change? Would be good to consult @Laurahohmann on this